src/elements/common/messages.js

```diff
@@ -1025,6 +1025,16 @@ const messages = defineMessages({
         description: 'Icon title for a Box item of type bookmark or web-link',
         defaultMessage: 'Bookmark',
     },
+    archive: {
+        id: 'be.archive',
+        description: 'Icon title for a Box item of type folder is an archive',
```

Comment:
is this supposed to be for a folder also or for a file ?

Reply:
Only folders - top level archive and a folder in archive (they have different icons), we'll not be distinguishing archived files in content explorer